### DISCUSSION TITLE
Integrate Torus Wallet on BackEnd

### DISCUSSION THREAD

**Discussion by thomazpro** [Web3Auth/Web3Auth#598] [Help]
Hello everyone, I would like to know if it is possible to instantiate a torus wallet in the back end, and how to leave it logged in this instance, because my backend will need an address to sign certain operations....thank you.

**Comment by himanshuchawla009** [comment]
Torus wallet is a browser-based wallet and cannot be used on the backend.

**Reply by thomazpro** [reply]
understood, thanks for the feedback, could you tell me if there is any possibility of using an ethereum address to sign transactions on the back end?

**Reply by thomazpro** [reply]
In my business I will have nfts in my wallet, and I want to allow users to be able to buy them automatically through conventional currencies such as the Real, which is the currency of my country Brazil.
